easybuild/tools/docs.py

```diff
@@ -727,13 +727,17 @@ def list_toolchains(output_format=FORMAT_TXT):
     """Show list of known toolchains."""
     _, all_tcs = search_toolchain('')
     all_tcs_names = [x.NAME for x in all_tcs]
-    tclist = sorted(zip(all_tcs_names, all_tcs))
 
-    tcs = dict()
-    for (tcname, tcc) in tclist:
+    # start with dict that maps toolchain name to corresponding subclass of Toolchain
+    tcs = dict(zip(all_tcs_names, all_tcs))
+
+    for tcname in sorted(tcs.keys()):
```

Comment:
This could be
`for tcname in sorted(tcs):`
but it's a bit a question of style (not that keys() is slower in Python 2 since it creates a full independent list instead of an iterator)